avocado/plugins/htmlresult.py

```diff
@@ -70,7 +70,12 @@ def passed(self):
         return self.json['pass']
 
     def pass_rate(self):
-        pr = 100 * (float(self.json['pass']) / float(self.json['total']))
+        total = self.json['total']
+        passed = self.json['pass']
+        if int(total) > 0:
```

Comment:
No doubt this code is sound, but enabling "nitpick mode" I'd write it as:

```
   total = float(self.json['total'])
   passed = float(self.json['pass'])
   if total > 0.0:
      pr = 100 * (passed / total)
   ...
```

Results in one less type conversion, and (nitpick++) a cleaner percentage calculation.

Question to self: do we check somewhere if `total >= passed` ?